src/components/Home/Alumni/Alumni.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import './alumni.css';
+import { companyListAlumni, universityList } from '../../data';
+
+const Alumni = () => {
+    const [companies, setCompanies] = useState(companyListAlumni);
+    const [universities, setUniversities] = useState(universityList);
+
+    const renderLink = (array) => {
+        return array.map((item, index) => (
+            <span key={item.name}>
+                <span className="alumni__highlight">
+                    {item.name}
+                </span>
+                {(index < array.length - 1 && array.length > 2) ? ', ' : ''}
+                {(index < array.length - 1 && array.length === 2) ? ' ' : ''}
+                {(index === array.length - 2) ? 'and ' : ''}
+            </span>
+        ));
+    }
+
+    return (
+        <div className="alumni section">
+            <h1 className="section__title">Our Alumni</h1>
+
+            <div className="text-container">
+                <div className="links-container">
+                    Competitive Programming has shaped the mindset of our members, refining them into the best version of themselves. Due to their own efforts, some of the companies of our alumni are 
+                    {companies.map((item) => (
+                        <span>
+                            <span className="alumni__highlight">
+                                {item.name}
+                            </span>, 
+                        </span>
+                    ))}
```

Comment:
Probably a good thing to separate this into a separate method to make this consistent with how universities are written in the code.